index.js

```diff
@@ -14,11 +14,10 @@
  * Module dependencies.
  */
 
-var os = require('os');
+var osName = require('os-name');
 
-var USER_AGENT = 'node/' + process.version
-  + ' (' + process.platform + ' ' + os.release()
-  + '; ' + process.arch + ')';
+var USER_AGENT = 'Node.js/' + process.version.slice(1)
```

Comment:
That new `process.release` Object should no longer be necessary that io.js and Node.js have merged.